lib/logstash/filters/translate.rb

```diff
@@ -114,11 +124,15 @@ class LogStash::Filters::Translate < LogStash::Filters::Base
 
   public
   def register
+    registering = true
     if @dictionary_path
       @next_refresh = Time.now + @refresh_interval
-      registering = true
       load_yaml(registering)
     end
+    if @dictionary_url
```

Comment:
I understand, you have ```dictionary_path``` or ```dictionary_url```, right? but not both? with this code is actually possible to have the two of them setup. I guess it would be better to make them exclusive. What do you think?